index.11ty.js

```diff
@@ -263,19 +263,15 @@ class Index extends Twitter {
 			<span class="tweets-primary-count-num">${this.renderNumber(tweetCount)}</span> tweet${tweetCount !== 1 ? "s" : ""}
 		</h2>
 
-		<form class="js" method="get" id="search-url">
-			<h2>Search for <label for="tweet-url">Tweet URL</label>:</h2>
+		<div class="js">
+			<h2>Search Tweets:</h2>
 			<div class="tweets-search">
-				<div class="lo" style="--lo-margin-h: 1em; align-items: center;">
-					<div class="lo-c" style="flex-grow: 100">
-						<input type="url" id="tweet-url" required placeholder="Tweet URL" style="width: 100%">
```

Comment:
@Accudio could we have a static search input, with a default form action using DuckDuckGo for example, that would be enhanced with PageFind, instead of a JS only search?

Reply:
Sounds good, however one snag is that it would require the build to know where the archive is deployed. Currently we don't have any metadata for this—would we be happy to add a `metadata.url` property or similar?

Otherwise that would be fine, although we can't enhance a search input using the Pagefind default UI we could replace it once Pagefind has been initialised.

Having a solution properly progressively-enhancing would be possible but a bit more work as we'd have to replace the UI library. This might be worth doing anyway, as the accessibility and flexibility of the default one isn't ideal, but I probably wouldn't be able to work on it for a while.

Reply:
You're right, we would need the URL somewhere.

I guess we can ship it like that, and open an issue for future enhancements.

Reply:
- #51